units/en/unit3/hands-on.mdx

```diff
@@ -0,0 +1,103 @@
+# Hands-on
```

Comment:
Maybe have an introduction explaining what's the goal of the hands on

Reply:
Yes, I think it'd be worth mentioning that we're building on top of the multi-view diffusion pipeline to go:
single image -> multi image -> splats

Reply:
Will do, thanks!